test/verify/check-login

```diff
@@ -143,6 +143,16 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
                                     "pam_unix\(cockpit:auth\): check pass; user unknown",
                                     "pam_succeed_if\(cockpit:auth\): requirement .* not met by user .*")
 
+        # Check that after logout we get back to /system
+        self.login_and_go("/users")
+        b.wait_in_text('#accounts-list', "user")
+        b.logout()
+        login("admin", "foobar")
+        with b.wait_timeout(10):
+            b.expect_load()
+        b.wait_present("#content")
+        b.wait_text('#content-user-name', 'Administrator')
```

Comment:
I don't think this proves anything -- `#content-user-name` is in in the navigation shell, i. e. it would appear always, regardless of what the current path is. May be better to check

    b.wait_js_cond('window.location.pathname == "/system"')

?

Reply:
🙈 Of course. I was experimenting with different field, but then copied this one from above and did not realize it does something different. Pushing fix.